src/Entity/Tags.php

```diff
@@ -0,0 +1,70 @@
+<?php
+
+namespace App\Entity;
+
+use Doctrine\Common\Collections\ArrayCollection;
+use Doctrine\Common\Collections\Collection;
+use Doctrine\ORM\Mapping as ORM;
+use ApiPlatform\Metadata\ApiResource;
+
+#[ORM\Entity]
+#[ApiResource]
+class Tags
+{
+    #[ORM\Id]
+    #[ORM\GeneratedValue]
+    #[ORM\Column]
+    private ?int $id = null;
+
+    #[ORM\ManyToMany(targetEntity: Job::class, inversedBy: 'tag')]
+    private Collection $job;
```

Comment:
Do you need this relation?

Reply:
Yes , a job have many Tag and a Tag have many job? I don't understand this question

Reply:
Ok but why bi directional?